src/metabase/util/string.clj

```diff
@@ -48,3 +50,8 @@
         "..."
         (when (< (+ end-limit start-limit) cnt)
           (subs s (- cnt end-limit) cnt)))))))
+
+(defn valid-uuid?
+  "True if the given string is formatted like a UUID"
+  [s] (try (java.util.UUID/fromString s) true
+           (catch Exception _e false)))
```

Comment:
nice one

Reply:
There's also `is-uuid-string?` in `metabase.test.util` which uses a regex implementation. Maybe these can be unified